x-pack/plugins/maps/public/classes/layers/blended_vector_layer/blended_vector_layer.ts

```diff
@@ -41,6 +40,10 @@ import { IVectorSource } from '../../sources/vector_source';
 
 const ACTIVE_COUNT_DATA_ID = 'ACTIVE_COUNT_DATA_ID';
 
+interface CountData {
```

Comment:
Not for this PR, but we should consider if generics for SyncContext and DataRequest would be useful to handle typing issues here. The datarequest-hooks are used for everything right now and a tighter typing might help readability. We might be able to remove the explicit casts, and it would also flag might help resolve the more "hidden" casts like https://github.com/elastic/kibana/blob/891342a76fce54192785f89e548c5144983e5ee5/x-pack/plugins/maps/public/actions/data_request_actions.ts#L197